WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -252,7 +251,7 @@ extension ReaderTabView {
         static let settingsButtonWidth: CGFloat = 56
 
         static let dividerWidth: CGFloat = .hairlineBorderWidth
-        static let dividerColor: UIColor = UIColor(light: .lightGray, dark: .darkGray)
+        static let dividerColor: UIColor = .neutral(.shade10)
```

Comment:
Looks like `.neutral(.shade10)` is the standard divider color, so maybe this could just be `.divider`?